lib/GADS/Schema/Result/Report.pm

```diff
@@ -304,12 +323,31 @@ sub create_pdf
         fg_color  => '#ffffff',
     };
 
-    $pdf->table(
-        data         => $fields,
-        header_props => $hdr_props,
-        border_c     => '#007c88',
-        h_border_w   => 1,
-    );
+    foreach my $topic (keys %$grouped_data) {
+        next if $topic eq 'Other';
```

Comment:
What is the reason for excluding the "other" topic?

Reply:
When this runs, for some reason, the placement of the 'Other' topic appears to be random each run - most runs it is rendered at the bottom, but for some it can be either the middle, or the top. I feel that said topic isn't as important as the named topics ('Other' denotes a field that has no topic) and as such should be rendered at the bottom.